src/ipc/types.ts

```diff
@@ -3,6 +3,8 @@ import { ItemCheckWidget, PriceCheckWidget } from '@/web/overlay/interfaces'
 export interface Config {
   configVersion: number
   leagueId?: string
+  privateLeagueName?: string
+  isPrivateLeague?: boolean
```

Comment:
and this will be a vue computed value in `League` service.

Reply:
Will make it so

Reply:
Actually this stores whether or not a private league is selected, it isn't computed state rather set state based on your selection. Should i make the computed value whether `leagueId === privateLeague.id` ?

Reply:
https://github.com/SnosMe/awakened-poe-trade/pull/464#discussion_r720834371

Reply:
going with "OR if it is not in the list of public leagues."